templates/cpp-template-default/proj.android/jni/Android.mk

```diff
@@ -11,9 +11,9 @@ LOCAL_MODULE := MyGame_shared
 
 LOCAL_MODULE_FILENAME := libMyGame
 
-LOCAL_SRC_FILES := hellocpp/main.cpp \
-                   ../../Classes/AppDelegate.cpp \
-                   ../../Classes/HelloWorldScene.cpp
```

Comment:
Got it! Nice work 
![aqua](https://user-images.githubusercontent.com/15274763/27434686-351a31be-5762-11e7-88e0-7adf8aa4a48f.jpg)